src/inference/tests/functional/CMakeLists.txt

```diff
@@ -46,6 +46,7 @@ ov_add_test_target(
             funcTestUtils
         INCLUDES
             $<TARGET_PROPERTY:inference_engine_obj,SOURCE_DIR>/src
+            ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
why?

Reply:
It is required to include new ov_extension_test.hpp.